src/main/groovy/pl/allegro/tech/build/axion/release/domain/VersionConfig.groovy

```diff
@@ -96,11 +96,15 @@ class VersionConfig {
     VersionWithPosition getRawVersion() {
         if (rawVersion == null) {
             ensureVersionServiceExists()
-            rawVersion = versionService.currentVersion(this, VersionReadOptions.fromProject(project))
+            rawVersion = versionService.currentVersion(this, getVersionReadOptions())
         }
         return rawVersion
     }
 
+    VersionReadOptions getVersionReadOptions() {
```

Comment:
if you opt to create a helper method, keep it private (VersionConfig is available as `scmVersion` extension in Gradle script)